clients/drcachesim/tools/opcode_mix.h

```diff
@@ -199,6 +199,9 @@ class opcode_mix_t : public analysis_tool_t {
     // For serial operation.
     worker_data_t serial_worker_;
     shard_data_t serial_shard_;
+    /* To guard the setting of isa_mode in dcontext.
```

Comment:
Better to use C++ style comments here. https://dynamorio.org/page_code_style.html#autotoc_md126 says "For C++ code, // comments are allowed." but doesn't disallow C-style comments. I wonder if it meant to.

Reply:
C-style are still useful for inline such as parameters `, /*param_name=*/ true,`.  But for whole-line `//` should be preferred for C++.

Reply:
Got it, changed it to `//`.